packages/experimental/artifact/src/types/tools.ts

```diff
@@ -66,8 +66,11 @@ export const ToolResult = Object.freeze({
 export const Tool = S.Struct({
   /**
    * Unique name.
+   * ^[a-zA-Z0-9_-]{1,64}$
    */
   name: S.String,
+  namespace: S.optional(S.String),
```

Comment:
add jsdoc for those pls

Reply:
ideally we'd replace the name with these; otherwise it's redundant -- need to discuss.